pkgs/tools/networking/grpcurl/default.nix

```diff
@@ -22,6 +22,7 @@ buildGoPackage rec {
     description = "Like cURL, but for gRPC: Command-line tool for interacting with gRPC servers";
     homepage = https://github.com/fullstorydev/grpcurl;
     license = stdenv.lib.licenses.mit;
+    platforms = with stdenv.lib.platforms; linux ++ darwin;
```

Comment:
`buildGoPackage` seems to set `platforms = lib.platforms.all` if no value is provided.
See:
https://github.com/zimbatm/nixpkgs/blob/686125dbf9e1b5e933d3e74a7843bd9b1939d272/pkgs/development/go-packages/generic/default.nix#L242

Reply:
```
nix-repl> grpcurl.meta.platforms
[ "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux" "armv7l-linux" "mipsel-linux" "i686-linux" "x86_64-linux" "powerpc64le-linux" "riscv32-linux" "riscv64-linux" "x86_64-darwin" "i686-darwin" "aarch64-darwin" "armv7a-darwin" ]
```

Reply:
and here is the latest hydra build:
https://hydra.nixos.org/eval/1569176?filter=grpcurl&compare=1569128&full=

Reply:
I should have checked better. Thanks! Will close the PR.

Reply:
No problem :)